releases/latest.releases.yaml

```diff
@@ -50,7 +50,7 @@ repositories:
   icub-models:
     type: git
     url: https://github.com/robotology/icub-models.git
-    version: v1.25.0
```

Comment:
It seems that this is bumping `icub-models` instead of `icub-firmware-models`, as noted by the CI. @pattacini

Reply:
Ops! I'll fix it at once.
icub-firmware-models also was at v1.25.0 and this tricked me.

Reply:
Fixed in #1281.